Request/index.ts

```diff
@@ -34,15 +34,26 @@ export namespace Request {
 			(value.header == undefined || RequestHeader.is(value.header))
 		)
 	}
+
 	export async function to(request: RequestLike): Promise<globalThis.RequestInit & { url: string }> {
 		const r = is(request) ? request : create(request)
+		const contentType = r.header.contentType
+		let headers
+		// If what is being sent is multipart/formdata, its previous content-type header
+		// needs to be removed in order for the new form-data boundary to be set correctly.
+		if (contentType?.split(";")[0] == "multipart/form-data") {
+			const newHeader = Object.fromEntries(Object.entries(r.header).filter(k => k[0] != "contentType"))
+			headers = newHeader
+		} else {
+			headers = r.header
+		}
```

Comment:
What about:
```
const header = contentType?.startsWith("multipart/form-data") ? (({ contentType: _; ...h }) => (h)(r.header) : r.header
```

Reply:
could not get this to work so tried to compromise